samples/apps/tinyRA/tinyra/run_tab.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import os
+from autogen import config_list_from_json
+from autogen import AssistantAgent, UserProxyAgent
+import tui
+
+msgid = None
+# get the first arg
+if len(sys.argv) > 1:
+    msgid = int(sys.argv[1])
+
+
+config_list = config_list_from_json("OAI_CONFIG_LIST")
+
+assistant = AssistantAgent("tinyra", llm_config={"config_list": config_list})
+user = UserProxyAgent(
+    "user",
+    code_execution_config={"work_dir": os.path.join(tui.DATA_PATH, "work_dir")},
+    human_input_mode="TERMINATE",
+    is_termination_msg=lambda x: "TERMINATE" in x.get("content", ""),
```

Comment:
note: next release of autogen will have `use_docker:True` as a default, so maybe a note in the readme might be worth it